examples/v1alpha3/nasjob-example-RL.yaml

```diff
@@ -67,7 +67,7 @@ spec:
                   - "RunTrial.py"
                   {{- with .HyperParameters}}
                   {{- range .}}
-                  - "--{{.Name}}={{.Value}}"
+                  - "--{{.Name}}=\"{{.Value}}\""
```

Comment:
why is this required?

Reply:
Here https://github.com/andreyvelich/katib/blob/cpu-example-rl-cifar10/examples/v1alpha3/NAS-training-containers/RL-cifar10/RunTrial.py#L13 I parse `architecture` and `nn_config` as a string. The format of architecture something like this `[[15], [86, 1], [7, 1, 0], [57, 1, 0, 0], [3, 0, 1, 1, 0], [42, 1, 1, 1, 1, 0], [17, 0, 0, 0, 1, 0, 0], [49, 0, 1, 1, 0, 0, 0, 1]]` and we need to send it as string. If we pass it without double quotes it fails in parsing.